tests/unittest_inference.py

```diff
@@ -549,8 +549,7 @@ def test_qqch(self):
         ast = parse(code, __name__)
         xxx = ast["xxx"]
         self.assertSetEqual(
-            {n.__class__ for n in xxx.inferred()},
-            {nodes.Const, util.Uninferable.__class__},
+            {n.__class__ for n in xxx.inferred()}, {util.Uninferable.__class__}
```

Comment:
The change in behavior in this test is due to the removal of 'import imp' in modutils.py. I'm having trouble understanding what 'qqch' is and this test has been part of astroid since the initial commit. If this adaptation of expected behavior is incorrect or perhaps the input code should be adapted to something else, let me know.